frontend/src/pages/user-profile/components/user-profile-sidebar/user-profile-sidebar.tsx

```diff
@@ -0,0 +1,29 @@
+import avatar from '#assets/img/avatar.png';
+import { useAppSelector } from '#libs/hooks/hooks.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+const UserProfileSidebar: React.FC = () => {
```

Comment:
Let's rename it to UserProfileInfo
```suggestion
const UserProfileInfo: React.FC = () => {
```

Reply:
done